html/docs/include/features.php

```diff
@@ -907,14 +907,80 @@
+
+<h2 name="usage">Usage</h2>
+
+<p>There are two approaches to start collecting the GC statistics with two different
+use-cases. The first is entirely via INI settings and the primary use-case is to
+collect statistics for indiviual CLI script runs (where GC is often an issue).</p>
+
+<p><code>php -dxdebug.gc_stats_enable=1 your_script.php</code></p>
+
+<p>If you wish to collect the garbage collection statistics for every script
+that you execute, you can set the [CFG:gc_stats_enable] INI setting on the system
+or directory-level. Be aware that activating the collection globally will produce
+an output file for *every* executed script, even if the garbage collector didn\'t
+run.</p>
+
+<p>The second approach to starting collection is to call the function
+<code>xdebug_start_gcstats</code> directly in your PHP script. This gives you
+more control over when collection is started.</p>
+
+<p>You can stop collection for both INI and function based approaches by
+calling <code>xdebug_stop_gcstats</code>.</p>
+
+<h2 name="output-tabular-text">Tabular Text (Human Readable)</h2>
+
+<p>The default (and only for now) output format of Garbage Collection statistics
+is a tabular human readable text output.</p>
+
+<p><code><pre>
+Garbage Collection Report
+version: 1
+creator: xdebug 2.6.0beta2-dev (PHP 7.2.0RC5)
+
+Collected | Efficiency% | Duration | Memory Before | Memory After | Reduction% | Function
+----------+-------------+----------+---------------+--------------+------------+---------
+    10000 |    100.00 % |  0.00 ms |       5539880 |       579880 |   -10.47 % | bar
+	10000 |    100.00 % |  0.00 ms |       5540040 |       580040 |   -10.47 % | Garbage::produce
+	 4001 |     40.01 % |  0.00 ms |       2563048 |       578968 |   -22.59 % | gc_collect_cycles
+</pre></code></p>
+
+<p>The header contains the version of the report and the version of XDebug that generated it.</p>
+
+<p>The table itself then contains one row for each garbage collection run each with 7 reported variables:</p>
+
+<ul>
+	<li><strong>Collected</strong> - The number of so called "roots" that have been garbage collected during this run. A "root" is either an PHP object (instance) or an array that is under observation by the Garbage Collector for potential cleanup.</li>
+	<li><strong>Efficiency%</strong> - Is the number of cleared roots divided by 10.000 - a magic number of "roots" when reached triggers PHPs internal garbage collector to run automatically.</li>
+	<li><strong>Duration</strong> - Denotes the duration in milliseconds that this garbage collection run took.</li>
+	<li><strong>Memory Before</strong> - Contains the memory as measured by <code>memory_get_usage()</code> before this GC run was activated.</li>
+	<li><strong>Memory After</strong> - Contains the memory as measured by <code>memory_get_usage()</code> after the GC run was finished.</li>
+	<li><strong>Reduction%</strong> - The percent reduction in memory due to this GC run.</li>
```

Comment:
Reduction with a negative number (as in the report), means to me that *more memory* was used. That's clearly not the case, so either we need to fix the code (remove the -), or the tables's header and comment here.